backend/src/appointment/database/repo/subscriber.py

```diff
@@ -71,8 +72,32 @@ def update(db: Session, data: schemas.SubscriberIn, subscriber_id: int):
     return db_subscriber
 
 
+def disable(db: Session, subscriber: models.Subscriber):
+    """Disable a given subscriber"""
+    # mark subscriber deleted = disable them
+    subscriber.time_deleted = datetime.datetime.now(datetime.UTC)
+    # invalidate session
+    # subscriber.minimum_valid_iat_time = datetime.datetime.now(datetime.UTC)
```

Comment:
This is giving me a weird error:

> sqlalchemy.exc.OperationalError: (MySQLdb.OperationalError) (1292, "Incorrect datetime value: 'xSoAjP5Wa5dyM51mj3ik+DSXLXqREIQAid+KUhryiFZ4yP/YPrUTDYoKnpwkoD3b' for column 'minimum_valid_iat_time' at row 1")
> [SQL: UPDATE subscribers SET time_updated=now(), time_deleted=%s, minimum_valid_iat_time=%s WHERE subscribers.id = %s]
> [parameters: (datetime.datetime(2024, 6, 1, 9, 46, 51, 624961, tzinfo=datetime.timezone.utc), 'xSoAjP5Wa5dyM51mj3ik+DSXLXqREIQAid+KUhryiFZ4yP/YPrUTDYoKnpwkoD3b', 7)]
> (Background on this error at: https://sqlalche.me/e/14/e3q8)

Obviously somehow sqlalchemy can't handle the encrypted datetime string? Maybe you have an idea on this @MelissaAutumn ? Commented it out for now.

Reply:
That is very odd. What is the minimum_valid_iat_time field in your database? It should be a VARCHAR(255) via https://github.com/thunderbird/appointment/blob/main/backend/src/appointment/migrations/versions/2024_01_09_1652-ad7cc2de5ff8_add_minimum_valid_iat_time_to_subscribers.py

Reply:
Oh good point, I'll investigate.

Reply:
You were totally right! Somehow I must have missed that migration and my minimum_valid_iat_time is still datetime 😅